src/ooni/utils.hpp

```diff
@@ -5,36 +5,44 @@
 #ifndef SRC_OONI_UTILS_HPP
 #define SRC_OONI_UTILS_HPP
 
+#include "src/ext/json/src/json.hpp"
+#include <GeoIP.h>
 #include <measurement_kit/common.hpp>
 #include <measurement_kit/http.hpp>
 #include <regex>
 #include <string>
 
+using Json = nlohmann::json;
```

Comment:
I understand why here you are using uppercase. I guess it's for uniformity with other places in MK where classes begin with uppercase, right? Yet, when I encountered `Json` in uppercase this stroke me, because I knew the class in `nlohmann::json` is lowercase. Given that, I'd keep it lowercase. Makes sense?